src/app/api/life.graphql

```diff
@@ -0,0 +1,18 @@
+query getLife($id: ObjectID!) {
+    getLife(id: $id) {
+        ...LifeData
+    }
+}
+
+
+query listLives {
+    listLives {
+        ...LifeData
+    }
+}
+
+mutation createLife($life: LifeCreateInput) {
+    createLife(life: $life) {
+        ...LifeData
+    }
+}
```

Comment:
EOF

Reply:
Fixed. Will include in next commit (part 5)